src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js

```diff
@@ -156,28 +161,50 @@ const PerformanceView = ({ data }) => {
             )}
           </Item>
           <Item title={t('Blocks generated')} icon="generatedBlocks">
-            <div className={styles.performanceValue}>{data.producedBlocks ?? '-'}</div>
+            <div className={styles.performanceValue}>{data?.producedBlocks ?? '-'}</div>
           </Item>
         </Box>
         <Box className={`${grid.col} ${grid['col-xs-4']} ${grid['col-md-4']} ${styles.column}`}>
-          <Item
-            icon="reward"
-            title={
-              <div>
-                <span>{t('Rewards')}</span>
-                <a>
-                  {t('See breakdown')}
-                  <Icon name="arrowBlueRight" />
-                </a>
+          <BoxContent className={`${styles.highlight} performance`}>
+            <div className={`${styles.content} ${styles.reward}`}>
+              <div className={`${styles.title} ${theme}`}>
+                <div>
+                  <span>{t('Rewards (LSK)')}</span>
```

Comment:
(LSK) This shouldn't be hardcoded, use {tokenSymbol} for dynamic rendering

Reply:
Fixed